src/main/java/de/chojo/repbot/config/elements/Botlist.java

```diff
@@ -1,33 +1,56 @@
 package de.chojo.repbot.config.elements;
 
-import java.util.ArrayList;
-import java.util.List;
+import de.chojo.jdautil.botlist.BotListConfig;
 
 @SuppressWarnings({"FieldMayBeFinal", "FieldCanBeLocal"})
 public class Botlist {
     private boolean submit;
-    private List<Long> guildIds = new ArrayList<>();
-    private String topGg = "";
-    private String discordBotsGg = "";
-    private String discordBotlistCom = "";
+    private String host = "0.0.0.0";
+    private int port = 8888;
+    private BotListConfig topGg = new BotListConfig("top.gg", "", "", 264445053596991498L, "", "");
+    private BotListConfig discordBotsGg = new BotListConfig("discord.bots.gg", "", "", 110373943822540800L, "", "");
+    private BotListConfig discordBotlistCom = new BotListConfig("discordbotlist.com", "", "", 450100127256936458L, "", "");
+    private BotListConfig botListMe = new BotListConfig("discordbotlist.com", "", "", 698637043240009738L, "", "");
```

Comment:
The empty strings, should they be set via configuration? I don't know how Jackson works, but does it allow pre setting some variables which will be overwritten on configuration load?

Reply:
Yes thats exactly how jackson works.
If the value is present in the json object, it will override the current default value of the